desktop/analytics/partner.js

```diff
@@ -22,4 +22,19 @@
       partner_slug: $(e.currentTarget).data('partner-slug')
     })
   })
+
+  $('.main-layout-container').on('click', '#partner2-contact .email-gallery', function (e) {
+    analytics.track('Click',{
+      partner_id: $(e.currentTarget).data('partner-id'),
+      label: "Contact gallery by email"
+    })
+  })
+
+  $('.main-layout-container').on('click', '#partner2-contact .partner2-website', function (e) {
```

Comment:
Following up on https://github.com/artsy/force/pull/1491#issuecomment-305590678, I remember we used to have app-specific namespace, like [`.partner2-content`](https://github.com/artsy/force/blob/cc8e8ff19b553f821768a42e3861bad7592433c1/desktop/apps/partner2/templates/index.jade#L13), to prevent conflict, but maybe that is also rendered client-side.

I think `.main-layout-container` should work too, given that the actual selector is unique. 👍